src/services/admin-api.ts

```diff
@@ -61,3 +49,37 @@ export interface ParkingRegistration {
   color: string;
   contact: string;
 }
+
+export interface ElevatorBooking {
+  id: number;
+  startTime: Date;
+  endTime: Date;
+  unit: number;
+  ownerType: string;
```

Comment:
Now that user type is added, is this still needed? Should it be replaced by user type?

Reply:
Good question! 

One of the requests was that owners can book for tenants in the case of moving in and out. Hence having a type here.

Reply:
Ahh I see, thanks!